docs/peer-dependencies.md

```
@@ -1,17 +1,21 @@
 # Shakapacker's Peer Dependencies
+
 ## Last updated for our 9.0.0 version — see lib/install/package.json
 
 To simplify peer dependencies while supporting both webpack & rspack, we decided to document the dependencies here instead of creating two separate npm packages.
 
 **Important Note**: Starting with v9, Babel dependencies are no longer included as peer dependencies. They will be installed automatically only if you're using Babel as your JavaScript transpiler.
 
 ## Essential for Rspack
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)